roles/container-engine/containerd/templates/config.toml.j2

```diff
@@ -39,3 +39,7 @@ version = 2
         [plugins."io.containerd.grpc.v1.cri".registry.mirrors."{{ registry }}"]
           endpoint = ["{{ ([ addr ] | flatten ) | join('","') }}"]
 {% endfor %}
+
+{% if 'extra' in containerd_config %}
```

Comment:
container_config is deprecated as https://github.com/kubernetes-sigs/kubespray/blob/bef1e628ac119f3f24b563307e7471cb4a9c5762/roles/kubernetes/preinstall/tasks/0020-verify-settings.yml#L311-L317

It is necessary to add containerd_extra_args or something instead.